### ISSUE TITLE
Web Demo is Broken

### ISSUE THREAD

**Issue by error454** [jovotech/jovo-starter-web-standalone#3]
I attempted to run the web demo at (https://www.jovo.tech/demos/starter-web-standalone/) on both Brave and Edge and receive the same error in both:

```
Uncaught Error: Unhandled error. ()
    at e.s.emit (index.umd.js:1)
    at SpeechRecognition.t.onerror (index.umd.js:1)
```

I made sure to accept microphone permissions and in brave I disabled shields.

**Comment by m-ripper** [comment]
Hey there,

first of all, thanks for the bug-report.

Brave and Edge do not *(fully)* support `SpeechRecognition` yet. 
For this reason, speech-recognition will only work in Chrome. 

We usually display a message that this demo can only be used in Chrome but there was a bug related to the browser- and feature-detection that caused the demo to not show the message. 

During the next days, we will publish the updated version that shows the previously mentioned message.

The error that is thrown by the SpeechRecognition occurs, because the API is not fully implemented for the browser.\
In Edge the error was `language-not-supported` no matter which language I used.

TLDR: To use thisdemo, I recommend using Chrome.
